scene/gui/rich_text_label.cpp

```diff
@@ -2216,6 +2220,35 @@ RichTextLabel::Item *RichTextLabel::_find_indentable(Item *p_item) {
 	return indentable;
 }
 
+List<RichTextLabel::Item *> RichTextLabel::_get_table_frames(ItemTable *p_table) const {
+	List<RichTextLabel::Item *> search_list;
+	List<RichTextLabel::Item *> frames;
+
+	// Add all children to search queue
```

Comment:
```suggestion
	// Add all children to search queue.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).